### PR TITLE
Added Syntax highlighting for docs.

### LINKED ISSUE #123: Syntax Highlighting in Docs
We can use `pygments` with pydocmd material theme for syntax highlighting.

### PR DESCRIPTION
**Pull request by josephbirkner**
Fixes #123

### CODE REVIEW

File: pydocmd.yml [hunk 1]
```diff
@@ -47,4 +47,8 @@ site_dir: docs
 # subdirectory of your project (eg. docs/), you may want to add the parent
 # directory here.
 additional_search_paths:
-- modules
\ No newline at end of file
+- modules
+
+# Extension for code highliting
```
Comment (nbasargin) [inline pydocmd.yml]: highliting -> highlighting?

File: docs/modules/ravestate_akinator/index.html [hunk 2]
```diff
@@ -464,14 +464,14 @@
...
                 
-                <pre><code>       _     _                              
-      | |   (_)              _              
- _____| |  _ _ ____  _____ _| |_ ___   ____ 
-(____ | |_/ ) |  _ \(____ (_   _) _ \ / ___)
-/ ___ |  _ (| | | | / ___ | | || |_| | |    
-\_____|_| \_)_|_| |_\_____|  \__)___/|_|    
-
-</code></pre>
+                <div class="codehilite"><pre><span></span>       <span class="n">_</span>     <span class="n">_</span>
```
Comment (nbasargin) [inline docs/modules/ravestate_akinator/index.html]: I liked the old version more :P

File: pydocmd.yml (hunk 1 above)
Comment (josephbirkner) [reply pydocmd.yml]: Oops

File: docs/modules/ravestate_akinator/index.html (hunk 2 above)
Comment (josephbirkner) [reply docs/modules/ravestate_akinator/index.html]: Generated code 🤷‍♂

**Pull request merged**